python/py-hid/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-hid
+version             1.0.5
+revision            0
+
+categories-append   comms
+
+license             MIT
+supported_archs     noarch
```

Comment:
from the .whl file on PyPI it looks like this port will install the same files on each macOS version  "platforms {darwin any}" could/should be added. 

Please check this in tour other submissions as well (wheel file contains "any_none").